config/routes.rb

```diff
@@ -8,6 +8,9 @@
 
     root 'projects#index'
 
+    # React Frontend-Framework
+    get '/react/settings', to: 'react/settings#index'
```

Comment:
hmm probably we should name this somehow different like jsAPI or clentAPI or whatever